src/rules/comment-no-empty/index.js

```diff
@@ -0,0 +1,31 @@
+import {
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "comment-no-empty"
+
+export const messages = ruleMessages(ruleName, {
+  rejected: "Unexpected empty comment",
+})
+
+export default function (actual) {
+  return (root, result) => {
+    const validOptions = validateOptions(result, ruleName, { actual })
+    if (!validOptions) { return }
+
+    root.walkComments(comment => {
+      // Check if the comment text is empty
```

Comment:
You'll want to [ignore inline SCSS comments](https://github.com/stylelint/stylelint/blob/master/src/rules/comment-empty-line-before/index.js#L57) here.